### PR TITLE
prs: 0.2.7 -> 0.2.8

### PR DESCRIPTION
**Pull request by dotlambda**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/security/prs/default.nix [hunk 1]
```diff
@@ -31,10 +32,16 @@ rustPlatform.buildRustPackage rec {
...
   '';
 
-  nativeBuildInputs = [ gpgme pkg-config python3 ];
+  nativeBuildInputs = [ gpgme installShellFiles pkg-config python3 ];
 
   buildInputs = [ dbus glib gpgme gtk3 libxcb ];
 
+  postInstall = ''
+    for shell in bash fish zsh; do
+      installShellCompletion --cmd prs --$shell <($out/bin/prs internal completions $shell --stdout)
```
Comment (dotlambda) [inline pkgs/tools/security/prs/default.nix:41]: @lilyball This generates `share/fish/vendor_completions.d/prs.fish` while it should be `share/fish/vendor_completions.d/_prs`.

**Comment by lourkeur** [comment]
Result of `nixpkgs-review pr 120311` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>prs</li>
  </ul>
</details>

**Review by bbjubjub2494 — APPROVED** [review]
completion appears to work

Capture d’écran de 2021-04-23 13-46-33

**Comment by dotlambda** [comment]
> completion appears to work

It does on bash and on fish.

**Pull request merged**